### PR TITLE
catalog-db-solr-sync playbook

### PR DESCRIPTION
**Pull request by jbrown-xentity**
This is a use case of a playbook for a ckan command via ansible. The playbook will skip if the package/dataset variable is not defined, and will try to ask for user input if not declared in-line.

Example usage:

    $pipenv run ansible-playbook ansible/actions/catalog-db-solr-sync.yml --extra-vars "db_solr_sync_package_name=youth-tobacco-survey-yts-data"

### CODE REVIEW

File: ansible/actions/catalog-db-solr-sync.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      when: db_solr_sync_package_name is undefined
+      register: pkg_name
+
+    - name: Get input
+      set_fact:
+        db_solr_sync_package_name: "{{ pkg_name.user_input }}"
+      when: db_solr_sync_package_name is undefined
+
+    - name: Run Sync Command
+      shell: "ckan search-index rebuild {{ db_solr_sync_package_name }}"
```
Comment (adborden) [inline ansible/actions/catalog-db-solr-sync.yml]: `command` is preferred to `shell`, unless you actually need shell syntax (`||`, `&&`, `|`, etc)

**Review by adborden — APPROVED** [review]
Looks good.

Just a tip, I _think_ search-index rebuild accepts a list of package names. So if you make the Ansible variable an array, you could do something like create a list of package names in YAML:
```yaml
---
# packages-for-rebuild.yml
db_solr_sync_package_name:
- pkg1
- pkg2
```

And then run it like `ansible-playbook actions/catalog-db-solr-sync.yml --extra-vars @packages-for-rebuild.yml`

**Comment by jbrown-xentity** [comment]
Actually, the search-index takes only an individual dataset:
```
root@catalog-harvester1tf:/home/ubuntu# ckan search-index rebuild --help
Usage: ckan search-index [options] Creates a search index for all datasets

    Usage:
      search-index [-i] [-o] [-r] [-e] rebuild [dataset_name]  - reindex dataset_name if given, if not then rebuild
                                                                 full search index (all datasets)
      search-index rebuild_fast                                - reindex using multiprocessing using all cores.
                                                                 This acts in the same way as rubuild -r [EXPERIMENTAL]
      search-index check                                       - checks for datasets not indexed
      search-index show DATASET_NAME                           - shows index of a dataset
      search-index clear [dataset_name]                        - clears the search index for the provided dataset or
                                                                 for the whole ckan instance

```
Also, see failing test:

```
root@catalog-harvester1tf:/home/ubuntu# ckan search-index rebuild superfund-sites marine-place-name
You must provide no more than 2 arguments
```

**Comment by adborden** [comment]
👍 thanks for checking.

**Pull request merged**